### PR TITLE
3.6.1 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-3.6.1-release](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-3.6.1-release)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.6.1 (2020-01-08)
+
+- Fixed the file drop component to accept multiple files when drag and dropped. [#92](https://github.com/blackbaud/skyux-forms/pull/92)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Is the component "file attachment" or "file drop"? 

"...to accept multiple files with drag and drop" or "...to accept multiple files when using drag and drop." 

Just trying to write around "when drag and dropped."

Comment (blackbaud-johnly) [reply CHANGELOG.md]: Looking at the PR, I'm pretty sure "file drop component," is correct, although the PR also updates the file attachment component and service, so it'd be good if Alex weighs in. As for the rewording, what do you think about `Fixed the file drop component to enable users to drag and drop multiple files.`?

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Yep that works for me. Sorry, I had a side convo on slack with @Blackbaud-DeniseCisneros , but I essentially said the same thing - it is indeed the file drop component. I'll go ahead and make that change.

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Done!

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-3.6.1-release](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-3.6.1-release)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-3.6.1-release](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-3.6.1-release)

**Pull request merged**